types/sfcookies/index.d.ts

```diff
@@ -14,9 +14,9 @@ export function bake_cookie(name: string, value: string, date?: Date): void;
 /**
  * Get the value of an cookie.
  * @param name Name of the cookie.
- * @returns cookie value.
+ * @returns cookie value or empty array if nothing.
  */
-export function read_cookie(name: string): string;
+export function read_cookie(name: string): string | [];
```

Comment:
```suggestion
export function read_cookie(name: string): string | unknown[];
```
it cannot be type-opaque (I understand the native code does not care here)